order.go

```diff
@@ -139,20 +139,30 @@ type OrderList struct {
 
 // OrderListParams is the set of parameters that can be used when listing orders.
 type OrderListParams struct {
-	ListParams   `form:"*"`
-	Created      *int64            `form:"created"`
-	CreatedRange *RangeQueryParams `form:"created"`
-	Customer     *string           `form:"customer"`
-	IDs          []*string         `form:"ids"`
-	Status       *string           `form:"status"`
+	ListParams        `form:"*"`
+	Created           *int64                         `form:"created"`
+	CreatedRange      *RangeQueryParams              `form:"created"`
+	Customer          *string                        `form:"customer"`
+	IDs               []*string                      `form:"ids"`
+	Status            *string                        `form:"status"`
+	StatusTransitions *StatusTransitionsFilterParams `form:"status_transitions"`
+}
+
+// StatusTransitionsFilterParams are parameters that can used to filter on status_transition when listing orders.
+type StatusTransitionsFilterParams struct {
+	StatusTransitions `form:"*"`
+	CanceledRange     *RangeQueryParams `form:"canceled"`
+	FulfilledRange    *RangeQueryParams `form:"fulfilled"`
+	PaidRange         *RangeQueryParams `form:"paid"`
+	ReturnedRange     *RangeQueryParams `form:"returned"`
 }
 
 // StatusTransitions are the timestamps at which the order status was updated.
 type StatusTransitions struct {
-	Canceled  int64 `json:"canceled"`
-	Fulfilled int64 `json:"fulfiled"`
-	Paid      int64 `json:"paid"`
-	Returned  int64 `json:"returned"`
+	Canceled  int64 `json:"canceled" form:"canceled"`
+	Fulfilled int64 `json:"fulfiled" form:"fulfilled"`
+	Paid      int64 `json:"paid" form:"paid"`
+	Returned  int64 `json:"returned" form:"returned"`
```

Comment:
And let's drop the `form` annotations from this struct — they're not needed.

Reply:
Done